assets/models/system/microsoft-phi-1-5/description.md

```diff
@@ -0,0 +1,107 @@
+## **Microsoft Phi-1.5**
```

Comment:
since we are adding it here. Do you want to update description to fit the standards defined by Tina in word doc

Reply:
can be ignored for now, since this is a preview model.